components/eamxx/scripts/check-hashes-ers

```diff
@@ -0,0 +1,171 @@
+    txt = readall(fn)
+    return greptxt(pattern, txt)
+
+###############################################################################
+def get_log_glob_from_atm_modelio(case_dir):
+###############################################################################
+    filename = case_dir / 'CaseDocs' / 'atm_modelio.nml'
+    ln = grep('diro = ', filename)[0]
+    run_dir = pathlib.Path(ln.split()[2].split('"')[1])
+    ln = grep('logfile = ', filename)[0]
+    atm_log_fn = ln.split()[2].split('"')[1]
+    id_ = atm_log_fn.split('.')[2]
+    return str(run_dir / f'e3sm.log.{id_}*')
+
+###############################################################################
+def get_hash_lines(fn):
+###############################################################################
+    rlns = run_cmd_no_fail(f'zgrep exxhash {fn}').splitlines()
+    lns = []
+    if len(rlns) == 0: return lns
+    for rln in rlns:
+        pos = rln.find('exxhash')
+        lns.append(rln[pos:])
+    return lns
+
+###############################################################################
+def parse_time(hash_ln):
+###############################################################################
+    return hash_ln.split()[1:3]
+
+###############################################################################
+def all_equal(t1, t2):
+###############################################################################
+    if len(t1) != len(t2): return False
+    for i in range(len(t1)):
+        if t1[i] != t2[i]: return False
+    return True
+
+###############################################################################
+def find_first_index_at_time(lns, time):
+###############################################################################
+    for i, ln in enumerate(lns):
+        t = parse_time(ln)
+        if all_equal(time, t): return i
+    return None
+
+###############################################################################
+def diff(l1, l2):
+###############################################################################
+    diffs = []
+    for i in range(len(l1)):
+        if l1[i] != l2[i]:
+            diffs.append((l1[i], l2[i]))
+    return diffs
+
+###############################################################################
+def check_hashes_ers(case_dir):
+###############################################################################
+    case_dir_p = pathlib.Path(case_dir)
+    expect(case_dir_p.is_dir(), f"{case_dir} is not a dir")
+
+    # Look for the two e3sm.log files.
+    glob_pat = get_log_glob_from_atm_modelio(case_dir_p)
+    e3sm_fns = glob.glob(glob_pat)
+    if len(e3sm_fns) == 0:
+        print('Could not find e3sm.log files with glob string {}'.format(glob_pat))
+        return False
+    e3sm_fns.sort()
+    if len(e3sm_fns) == 1:
+        # This is the first run. Exit and wait for the second
+        # run. (POSTRUN_SCRIPT is called after each of the two runs.)
+        print('Exiting on first run.')
```

Comment:
Minor, but maybe we could print more descriptive messages, so that someone looking at TestStatus.log would understand what this message refers to. Something like
```python
print ("[check-hashes-ers postrun script log] <rest of msg here>")
```
For ppl familiar with this postrun script, the messages are already enough, but the avg user may not understand what's going on.

Reply:
@bartgol , all postrun script output goes to a separate log, `$RUNDIR/postrun.external.log.$testid`. Does that change your thoughts on this?

Reply:
So it doesn't end up in TestStatus.log? Then I suppose it's fine.